vast_csi/plugins/base.py

```diff
@@ -68,6 +68,7 @@ def wrapper(self, request, context):
             params = {fld.name: value for fld, value in request.ListFields()}
             # secrets are not logged and not the part of function signature.
             secrets = params.pop("secrets", {})
+            host_encryption_secret = params.pop("host_encryption_secret", {})
```

Comment:
As we discussed in slack not possible to provide multiple secrets.
You might want to add `passphrase` to "main" secrets and take it from "secrets" 

passprase = secrets.pop("passprase") ...